kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ResourcesUtil.java

```diff
@@ -176,6 +176,9 @@ public static <T extends HasMetadata> LocalRef<T> toLocalRef(T ref) {
     }
 
     public static String group(HasMetadata resource) {
+        if (!resource.getApiVersion().contains("/")) {
```

Comment:
I found that using `toLocalRef` on a Secret failed while writing the unit tests, for the built in types the `apiVersion` is just the string `v1` and if you get the `group` via `HasMetadata#getGroup()` it's an empty string. So I've gone with empty string too.

Reply:
OK, that makes sense. Kube uses the empty group for some core APIs (but in some contexts they get called the `core` group, so it's not always clear what to expect).

Reply:
Maybe add a comment?

Reply:
done